content/en/docs/armory-admin/dinghy-enable.md

```diff
@@ -270,9 +274,10 @@ to be able to reach back out to your GitLab installation; ensure that
 connectivity works as well.
 
 ### Custom branch configuration
-*Note: this feature requires Armory 2.5.6 or above.*
 
-By default, Dinghy will use the Master branch in your repository. If you wish to use a different default branch for your repository, this can be configured using the `repoConfig` tag in your yaml configuration.
+> Configuring a custom branch is required if you are using a repo that no longer uses `master` as its primary branch.
+
+By default, Dinghy will use the `master` branch in your repository. If you wish to use a different default branch for your repository, this can be configured using the `repoConfig` tag in your yaml configuration.
```

Comment:
With newly created Repos now using "main" as the default branch, users will need to set "main" as a repo to be monitored by using the `repoConfig` tag